tests/examples/shims/client.lua

```diff
@@ -71,6 +81,20 @@ function client.gen_fake(args)
         return nil
     end
 
+    function ret:get_icon(_)
+        return require("beautiful").awesome_icon
+    end
+
+    function ret:raise()
+        --TODO
+    end
+
+    function ret:lower()
+        --TODO
+    end
```

Comment:
The functions are called by some of the newer tests, but the behavior is not (yet) required. To keep things simple, I would rather not implement z-index support in this commit.